libs/plugin-types/index.d.ts

```diff
@@ -194,6 +194,70 @@ export interface PluginData {
   getSharedPluginDataKeys(namespace: string): string[];
 }
 
+/**
+ * Comments allow the team to have one priceless conversation getting and
+ * providing feedback right over the designs and prototypes.
+ */
+export interface Comment {
+  /**
+   * The `user` that has created the comment.
+   */
+  readonly user: User;
+
+  /**
+   * The `date` the comment has been created.
+   */
+  readonly date: Date;
+
+  /**
+   * The `text` for the commentary. The owner can modify the comment.
+   */
+  text: string;
+
+  /**
+   * Remove the current comment from its comment thread.
```

Comment:
Can the owner delete a comment or it could be done by anyone?  We explicitly say the owner is the only one who can modify a comment but we say nothing about delete it.

Reply:
Yes, only the owner.